import mysql.connector

connection = mysql.connector.connect(
    host="109.206.169.221",
    user="seschool_01",
    password="seschool_01",
    database="seschool_01"
)

cursor = connection.cursor()

Books = [
(40000,"Богатый Папа"),
(13000,"Бедный Папа"),
(96000,"Гарри Поттер"),
(51000,"Баба Яга"),
(74000,"Каша из Топора"),
(41000,"Колобок"),
(21000,"Ревизор"),
(35000,"Психология"),
(33000,"Собачье Сердце"),
(15000,"КотоПёс")]

query_B = 'INSERT INTO Books (Price, Name) VALUES(%s);'
cursor.executemany(query_B, Books)

WareHouse = [
(1, 15),
(2, 2313),
(3, 140),
(4, 250),
(5, 370),
(6, 200),
(7, 400)]

query_ware = 'INSERT INTO WareHouse VALUES(%s);'
cursor.executemany(query_ware, WareHouse)

POST = [
(1, "Электрик"),
(2, "Сантехник"),
(3, "Консультант"),
(4, "Менеджер")
]


query_P = 'INSERT INTO Post VALUES(%s);'
cursor.executemany(query_P, POST)

Provider = [
(1, 4000,"ООО Клавиатура"),
(2, 1300,"ИП СЕМЁНОВ"),
(3, 9600,"ИП Мониторов"),
(4, 5100,"ООО Яга"),
(5, 7400,"ИП Топор"),
(6, 4100,"ИП Колобок"),
(7, 2100,"ООО Ревизор"),
(8, 3500,"ООО Психология"),
(9, 3300,"ООО Собачье Сердце"),
(10, 1500,"ИП КотоПёс")]

query_Provider = 'INSERT INTO Provider (id_books, price, name) VALUES(%s);'
cursor.executemany(query_Provider, Provider)

AuthorToBook = [
(1, 2),
(2, 2),
(3, 3),
(4, 5),
(5, 4),
(1, 3),
(2, 1),
]

query_AB = 'INSERT INTO AuthorsToBooks VALUES(%s);'
cursor.executemany(query_AB, AuthorToBook)


ShopToBooks = [
(1, 2),
(2, 2),
(3, 3),
(4, 5),
(5, 4),
(1, 3),
(2, 1),
]

query_SB = 'INSERT INTO ShopToBooks VALUES(%s);'
cursor.executemany(query_SB, ShopToBooks)


connection.commit()
connection.close()



# Traceback (most recent call last):
  File "C:\Users\DiEnly\PycharmProjects\pythonProject3\main.py", line 3, in <module>
    connection = mysql.connector.connect(
                 ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\DiEnly\PycharmProjects\pythonProject3\.venv\Lib\site-packages\mysql\connector\pooling.py", line 322, in connect
    return CMySQLConnection(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\DiEnly\PycharmProjects\pythonProject3\.venv\Lib\site-packages\mysql\connector\connection_cext.py", line 151, in __init__
    self.connect(**kwargs)
  File "C:\Users\DiEnly\PycharmProjects\pythonProject3\.venv\Lib\site-packages\mysql\connector\abstracts.py", line 1426, in connect
    self._open_connection()
  File "C:\Users\DiEnly\PycharmProjects\pythonProject3\.venv\Lib\site-packages\mysql\connector\connection_cext.py", line 340, in _open_connection
    raise get_mysql_exception(
mysql.connector.errors.InterfaceError: 2026 (HY000): SSL connection error: error:00000000:lib(0)::reason(0)
